package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "buying-catalogue-components",
-  "version": "3.3.13",
+  "version": "3.4.0",
```

Comment:
I don't know, I have worked on order-form and public-browse in order to make the 'accessibility-audit' branch work but these changes may well break other projects.

I am trusting in the automatic testing to prevent breaking changes being automatically brought into other project and will fix as I go. I realise that it will be a priority for me to do so.

Reply:
To be on the safe side let's bump up the major version then, so we'll go to `4.0.0`. That also keeps us nicely lined up with the NHSD component version (although that's not essential).